shopping-page.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>GitShop</title>
+    <link rel="stylesheet" href="./stylecss/shopping-page.css">
+</head>
+<body>
+   
+    <section class="page-container">
+    <img class="shop-img" src="./image/shopping-bags.jpg" alt="shopping |E-Commerce">
+    <div class="content">
+        
+        <span id = "titles">Our Vision <br> </span> <br />
+        <span  id ="term">" To build a place where people can come to find and discover anything <br>they  might want to buy online." <br></span> <br />
+   
+        <span id = "titles" >Our Mission <br> </span> <br />
+        <span  id ="term" > "Our mission is to continually raise the bar of the customer<br> experience by using the internet and technology to help <br>consumers find, discover and buy anything, and empower<br> businesses and content creators to maximise their success."</span
```

Comment:
It looks like you use `id = "titles"` and `id = "term"` for two elements - `id` is required to be unique on the page (meaning there is only one element that uses a given `id`), so it might be a good idea to switch those to use `class` instead of `id`, since the browser allows for multiple elements with the same class
For examle:
`<span class="titles">`
`<span class="term">`